interfaces/portal/package.json

```diff
@@ -92,15 +92,15 @@
     "eslint-plugin-regexp": "^2.9.0",
     "eslint-plugin-simple-import-sort": "^12.1.1",
     "eslint-plugin-sort-class-members": "^1.21.0",
-    "globals": "^16.2.0",
-    "jasmine-core": "~5.8.0",
+    "globals": "^16.3.0",
+    "jasmine-core": "~5.10.0",
     "karma": "~6.4.4",
     "karma-chrome-launcher": "~3.2.0",
     "karma-coverage": "~2.2.1",
     "karma-jasmine": "~5.1.0",
     "karma-jasmine-html-reporter": "~2.1.0",
-    "knip": "^5.62.0",
-    "ng-extract-i18n-merge": "^2.15.1",
+    "knip": "^5.63.0",
+    "ng-extract-i18n-merge": "^3.0.0",
```

Comment:
@RubenGeo I don't think you've run `ng update ng-extract-i18n-merge` for this right?

According to their README there are some 'migration' steps that will be necessary (and will be run automatically) and require manual approval/changes.